README.md

```diff
@@ -16,43 +16,50 @@ Installation requires the following steps:
+    * In `Sign-on URL` enter some URL, for example `http://alo365collector.com`. **Note**, it is not used anywhere within your subscription.
 
-1. After application is created select it and make a note of `Application Id`, for example, `a261478c-84fb-42f9-84c2-de050a4babe3`
+1. After application is created select `All apps`, click on the application name that was created and make a note of the `Application ID`, for example, `a261478c-84fb-42f9-84c2-de050a4babe3`
 
 ## Set Up the Required Active Directory Security Permissions
 
-1. On the `Settings` panel and select `Required permissions` and click `+Add`
+1. On the `Settings` panel under the newly created Application, select `Required permissions` and click `+ Add`
 1. Hit `Select an API` and chose `Office 365 Management APIs`, click `Select`
-1. In `Application permissions` select `Read service health information for your organization`, `Read activity data for your organization`, `Read threat intelligence data for your organization` and `Read activity reports for your organization`. Click `Select` and `Done` buttons.
-1. On `Required permissions` panel click `Required permissions` button and confirm the selection. **Note**, only AD tenant admin can grant permissions to an Azure AD application.
-1. On the `Settings` panel of the application and select `Keys`.
-1. Enter key `Description` and `Duration` and click `Save`. **Note**, please save the key value, it is needed later during template deployment.
-1. Save the `Application ID` and `Service Principal Id` for use below. To get the `Service Principal Id`, navigate to the `Registered App` blade, 
-click on the link under `Managed application in local directory`.  Then click `Properties`.  The `Service Principal Id`
-is labeled `Object ID` on the properties page.  **Caution** This is not the same `Object ID` listed in the `Properties` blade reached 
-by clicking `Settings` or `All Settings` from the `Registered app`.  It is also not the `Object ID` shown on the `Registered app`
-blade itself.
+1. In `Application permissions` select `Read service health information for your organization`, `Read activity data for your organization`, `Read threat intelligence data for your organization` and `Read activity reports for your organization`. Click `Select` and then `Done`.
+1. Click the `Grant Permissions` button and confirm by clicking `Yes`. **Note:** only AD tenant admin can grant permissions to an Azure AD application.
+1. On the `Settings` panel of the application, select `Keys`.
+1. Enter key `Description` and set `Duration` to `Never expires` then click `Save`.
+**Note:** please save the key value, it is needed later during template deployment.
+1. Get the `Service Principal Id` associated with the application by navigating back to the `Registered App` blade, click on the link under `Managed application in local directory` then click `Properties`. The `Service Principal Id`
+is labeled as `Object ID` on the properties page.
```

Comment:
Should this be "ID" instead of "Id"? - it's changed like that in other places.

Reply:
Made the change.